conftest.py

```diff
@@ -17,6 +60,22 @@ def pytest_configure(config):
 
 
 def pytest_runtest_setup(item):
+    # https://buildkite.com/docs/apis/rest-api/test-engine/quarantine#list-quarantined-tests
+    # Buildkite Test Engine marks unreliable tests as muted and triages them out to owning teams to improve.
+    # We pull this list of tests at the beginning of each pytest session and add soft xfail markers to each
+    # quarantined test.
+    try:
+        if buildkite_quarantined_tests():
+            scope = item.location[0]
+            name = item.location[2]
+
+            test = TestId(scope, name)
+
+            if test in buildkite_quarantined_tests():
+                item.add_marker(pytest.mark.xfail(reason="Test muted in Buildkite.", strict=False))
```

Comment:
Is it possible to print or log this to the buildkite console?

Reply:
Sure - I think it makes the most sense to do it by playing around with how pytest does its reporting (for example, `-rxX` will print a summary of failing fail and passing xfail tests as the end). I'll handle that in a  separate PR.